PWGCF/JCorran/TableProducer/JCatalyst.cxx

```diff
@@ -149,18 +149,18 @@ struct JCatalyst {
       else if (trackingMode == 1 && !track.isGlobalTrackSDD())
         continue;
 
-      Double_t pt = track.pt();
+      float pt = track.pt();
       if (pt < ptmin || pt > ptmax)
         continue;
 
       Int_t ch = track.sign();
       if (charge != 0 && charge * ch < 0)
         continue;
 
-      Double_t eta = track.eta();
-      Double_t phi = track.phi();
+      float eta = track.eta();
+      float phi = track.phi();
```

Comment:
In L.169 you are basically replicating the whole tracks table in memory. This will have an impact in the memory consumption of your task. Is that really needed? Wouldn't it be enough to produce only the additional columns?